pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -957,9 +957,10 @@ type Timer struct {
 //BEGIN Channel --------------------
 
 type Channel struct {
-	Type   string         `xml:"type,attr"`
-	Source *ChannelSource `xml:"source,omitempty"`
-	Target *ChannelTarget `xml:"target,omitempty"`
+	Type    string          `xml:"type,attr"`
```

Comment:
Can you put the virtwrap API change into a separate commit please?

Reply:
We have dropped the virtwrap API changes. It is better to let libvirt decide the address parameters.